### DISCUSSION TITLE
Corporate mail issued by the university is not suitable.

### DISCUSSION THREAD

**Discussion by PJOOONK** [community/community#144087] [GitHub Education] [answered]
### Hi everyone,

I enter the email that the university gave me. The mail automatically identifies the university with this domain, but returns the error: "We require applicants to use a school-issued email address to apply".

**Comment by davevad93** [comment, marked as answer]
Hi @PJOOONK ,

Have you added and verified your academic email? If not go to your [emails settings](https://github.com/settings/emails), add the academic email, verify it and set it as your primary email until your request for the `Student Developer Pack` has been approved, then you can switch back to your personal email as the primary GitHub email.
